### DISCUSSION TITLE
Selain TablePlus?

### DISCUSSION THREAD

**Discussion by Rizaziz8** [teamalgoritma/community#505] [Q&A] [answered]
Adakah database yang mudah di pakai selain dari TablePlus yang bisa digunakan bersama Jupyter notebook?

**Comment by onlyphantom** [comment, marked as answer]
Kalau database nya terbentuk sqlite, bisa juga langsung pakai VSCode saja. Sangat gampang sekali, dan tidak harus download aplikasi lain. 

ipynb file nya langsung saja dibuka di VSCode (VSCode has support for jupyter notebook / ipynb).

[SQLite in VSCode, Youtube explanation (7 mins)](https://youtu.be/VKg1Dnz7GN0)
